rocky/reports/views/base.py

```diff
@@ -58,16 +58,16 @@ def build_breadcrumbs(self):
         return breadcrumbs
 
     def get_current(self):
-        return self.build_breadcrumbs()[: self.current_step]
+        return self.build_breadcrumbs()[: self.breadcrumbs_step]
 
     def get_previous(self):
-        return self.build_breadcrumbs()[self.current_step - 2]["url"]
+        return self.build_breadcrumbs()[self.breadcrumbs_step - 2]["url"]
```

Comment:
make sure we always return at least the first page, or possibly False when no previous step is available.

Reply:
Yes, it is already like that. `breadcrumbs_step` has a default int of 1 and also if no step is added to the view and no breadcrumbs has been defined, you will see no breadcrumbs in the page.

Reply:
Ahh yes, now I see it. for get_previous to work there must be at least 2 breadcrumbs.